apis/zoho_api.py

```diff
@@ -10,7 +10,7 @@
 from requests import RequestException
 import zcrmsdk as zoho_crm
 import pytz
-from setup import CONFIG_FILE
+from ..setup import CONFIG_FILE
```

Comment:
why `..`?

Reply:
It's relative path
Because of it we can import higher level files